studio/components/interfaces/Database/Backups/BackupsEmpty.tsx

```diff
@@ -0,0 +1,14 @@
+import { IconInfo } from '@supabase/ui'
+
+const BackupsEmpty = () => {
+  return (
+    <div className="block w-full rounded border border-gray-400 border-opacity-50 bg-gray-300 p-3">
+      <div className="flex space-x-3">
+        <IconInfo size={20} strokeWidth={1.5} />
+        <p className="text-sm">No backups created yet. Check again tomorrow.</p>
```

Comment:
I found this a little confusing when I was browsing my test project. Do we know when backups are done? Nightly SG time or some other time? Would be nice to provide any extra info we have there.

Reply:
checking with angelico on the details for scheduled backups

Reply:
scheduled backups happen at 12 midnight based on the project's region - i'll add that message here to make it clearer

Reply:
![image](https://user-images.githubusercontent.com/19742402/175496530-71563292-9117-43c5-bdd0-258fb5e474be.png)

Reply:
Nice!